services/web/server/tests/unit/with_dbs/conftest.py

```diff
@@ -239,41 +239,56 @@ async def redis_client(loop, redis_service):
 
 
 @pytest.fixture()
-async def socketio_url(client) -> str:
-    SOCKET_IO_PATH = "/socket.io/"
-    return str(client.make_url(SOCKET_IO_PATH))
+def socketio_url(client) -> Callable:
+    def create_url(client_override: Optional = None) -> str:
+        SOCKET_IO_PATH = "/socket.io/"
+        return str((client_override or client).make_url(SOCKET_IO_PATH))
+
+    yield create_url
 
 
 @pytest.fixture()
-async def security_cookie(client) -> str:
-    # get the cookie by calling the root entrypoint
-    resp = await client.get("/v0/")
-    payload = await resp.json()
-    assert resp.status == 200, str(payload)
-    data, error = unwrap_envelope(payload)
-    assert data
-    assert not error
+async def security_cookie(client) -> Callable:
+    async def creator(client_override: Optional = None) -> str:
+        # get the cookie by calling the root entrypoint
+        resp = await (client_override or client).get("/v0/")
+        payload = await resp.json()
```

Comment:
why not using ``assert_response`` from ``pytest-simcore`` helpers?

Reply:
done